indico/modules/events/notification_templates/models/templates.py

```diff
@@ -0,0 +1,137 @@
+# This file is part of Indico.
```

Comment:
Does it really make sense to add a whole module just for the base model? I'd rather use `indico/modules/events/notification_templates.py` - it's not really a module if it has nothing besides some model.